_config.yml

```diff
@@ -28,7 +28,11 @@ exclude: ["Gemfile",
           "scripts/",
           ".gitignore",
           ".sass-cache/",
-          ".idea/"]
+          ".idea/",
+          "README.md",
+          ".ruby-version",
+          "_includes/site",
+          "_sass"]
```

Comment:
These `.scss` files look like they might be used?

Reply:
`scss` is generally excluded as this need not be included in the generated `_site`. Also, I have tested the changes on my local and there are no changes in the layout or the content display format after excluding these files.